coderadar-test/src/main/asciidoc/admin.adoc

```diff
@@ -12,89 +12,85 @@ v{version}, {date}
-| coderadar.CommitAnalyzer.commits.*
-| Count and throughput of commits having been analyzed.
+==== Configuring CORS
 
-| coderadar.CommitAnalyzer.files.*
-| Count and throughput of files having been analyzed.
+CORS can be enabled or disabled with the following property:
 
-| coderadar.CommitMetaDataScanner.commits.*
-| Count and throughput of commits having been scanned for metadata.
+`coderadar.cors.enabled=true`
 
-| coderadar.CommitToFileAssociator.commits.*
-| Count and throughput of commits whose files have been associated with it.
+==== Configuring logging levels
 
-| coderadar.CommitToFileAssociator.files.*
-| Count and throughput of files having been associated with a commit.
+The logging levels can be adjusting with the following properties:
 
-| coderadar.FileMetadataScanner.commits.count.*
-| Count and throughput of commits whose files have been scanned for metadata.
+`logging.level.io.reflectoring.coderadar=DEBUG`
+`logging.level.org.reflections=ERROR`
 
-| coderadar.FileMetadataScanner.files.*
-| Count and throughput of files having been scanned for metadata.
+==== Configuring the log file
 
-| coderadar.jobs.acquisitionConflicts.*
-| Count and rate of conflicts between server nodes who tried to acquire the same
-  job for processing. It's OK if this happens and the `coderadar.jobs.inProgress`
-  counter stays at approximately the number of coderadar slaves in the cluster.
-  If the `coderadar.jobs.inProgress` counter falls significantly below the number
-  of coderadar slaved in the cluster, they are probably contesting too much for
-  the jobs waiting in queue.
+The log file's path can be adjusted with the following property:
 
-| coderadar.jobs.failed.*
-| Count and rate of failed jobs.
+`logging.file=coderadar.log`
 
-| coderadar.jobs.finished.*
-| Count and throughput of successfully finished jobs.
+==== Configuration the database connection
 
-| coderadar.jobs.inProgress
-| Current number of jobs that are in progress over all coderadar servers in the cluster.
+The credentials for the Neo4j database instance can be configured with the following properties:
 
-| coderadar.jobs.waiting
-| Current number of jobs that are waiting for processing over all coderadar servers in the cluster.
+`spring.data.neo4j.username=neo4j`
+`spring.data.neo4j.password=neo3j`
 
-|===
\ No newline at end of file
+=== Starting Coderadar
+
+Starting Coderadar requires a running Neo4j instance on port 7687.
+Coderadar works with Neo4j version 3.5.* and requires the APOC plugin to be installed.
+The easiest way to quickly configure a local Neo4j instance is by using Neo4j Desktop.
+
+Once started, the Coderadar REST-API will be available on port 8080.
+For more information on it, have a look at the REST-API documentation:
+
+https://reflectoring.github.io/coderadar/current/docs/restapi.html
+
+=== Starting the Coderadar UI
+
+To start the Coderadar UI, you must first build it. This requires having NPM installed, going to the `coderadar-ui` directory and
```

Comment:
Suggestion: instead of **you must first build it** write **you have to build it first**.

Reply:
Done